src/app/services/proxy/user-proxy.service.ts

```diff
@@ -48,4 +48,10 @@ export class UserProxy {
   createFrilansFinans(userId, bankAccount) {
     return this.apiCall.post('users/' + userId + '/frilans-finans', bankAccount);
   }
+
+  resetPassword(email_or_phone: string) {
```

Comment:
_minor_ think we've gone for camelCased variable names (the API though is all underscores).